packages/ui/v2/core/form/date-range-picker/styles.css

```diff
@@ -0,0 +1,150 @@
+.react-daterange-picker > .react-daterange-picker__wrapper {
+  padding-top: 0.625rem;
+  padding-bottom: 0.625rem; 
+  padding-left: 0.75rem;
+  padding-right: 0.75rem; 
+  font-size: 0.875rem;
+  line-height: 1.25rem; 
+  border-radius: 0.375rem; 
+  border-color: #E5E7EB; 
+  width: 360px;
+}
+
+.react-daterange-picker.react-daterange-picker--disabled {
+  background-color: transparent;
+}
+
+.react-daterange-picker.react-daterange-picker--disabled > .react-daterange-picker__wrapper {
+  background-color: #F3F4F6;
+}
+
+.react-daterange-picker > .react-daterange-picker__wrapper:focus-within,
+.react-daterange-picker > .react-daterange-picker__wrapper:focus-within:hover {
+  border-color: #111827;
+  border-width: 2px;
+}
+
+
+.react-daterange-picker > .react-daterange-picker__wrapper input {
+  padding-top: 0.125rem;
+  padding-bottom: 0.125rem; 
+  margin-top: 0.125rem;
+  margin-bottom: 0.125rem; 
+  height: auto; 
+  border-radius: 0.125rem; 
+}
+
+.react-daterange-picker__calendar.react-daterange-picker__calendar--open {
+  width: 360px;
+}
+
+.react-daterange-picker__calendar > .react-calendar{
+  padding: 0.5rem; 
+  border-radius: 0.375rem; 
+  border-width: 1px; 
+  border-color: #E5E7EB; 
+  width: 360px;
+}
+
+.react-calendar__navigation > .react-calendar__navigation__arrow {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 6px;
+}
+
+.react-calendar__navigation > .react-calendar__navigation__arrow.react-calendar__navigation__prev2-button,
+.react-calendar__navigation > .react-calendar__navigation__arrow.react-calendar__navigation__next2-button {
+  display: none;
+}
+
+.react-calendar__navigation > .react-calendar__navigation__label {
+  display: flex; 
+  padding-left: 0.5rem;
+  padding-right: 0.5rem; 
+  color: #111827; 
+  order: -9999; 
+  justify-content: flex-start; 
+  align-items: center; 
+  font-size: 16px;
+}
+
+.react-calendar__navigation > .react-calendar__navigation__arrow {
+  text-align: center;
+  color: #6b7280;
+}
+
+.react-calendar__month-view__weekdays__weekday > abbr {
+  font-weight: 500; 
+  text-decoration: none; 
+  font-size: 12px;  
+}
+
+.react-calendar__month-view__days {
+  padding: 0.25rem; 
+  gap: 0.25rem; 
+  font-size: 14px;
+}
+
+button.react-calendar__tile.react-calendar__month-view__days__day {
+  flex : 0 0 13.25% !important;
```

Comment:
this property is added via JS in the form of inline styling, hence using `!important` is the only way to over-ride it